registry/Scripts/script.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash
```

Comment:
FYI, you can wrap the lines you want to protect using `set -x` and `set +x` instead of disabling debug completely